src/ssr-template-helper.js

```diff
@@ -105,10 +116,17 @@ export class SsrTemplateHelper {
         typeof data['html'] === 'string',
         'Server side html response must be defined'
       );
-      renderTemplatePromise = this.templates_.findAndSetHtmlForTemplate(
-        element,
-        /** @type {string} */ (data['html'])
-      );
+      renderTemplatePromise = this.viewer_.isTrustedViewer().then(trusted => {
```

Comment:
Looks like we may want to move the `viewer.isTrusted().then(assert(trusted))` into a separate function - e.g. `assertTrustedViewer():Promise`?

Reply:
Sure, I'll do that now